supervisor/addons/addon.py

```diff
@@ -87,6 +87,10 @@ def __init__(self, coresys: CoreSys, slug: str):
         self.instance: DockerAddon = DockerAddon(coresys, self)
         self.state: AddonState = AddonState.UNKNOWN
 
+    def __repr__(self) -> str:
```

Comment:
Shouldn't this be in `AddonModel` instead?

Reply:
No, they own by Addon and Store